src/main/java/net/dries007/tfc/objects/inventory/capability/IFluidTankCallback.java

```diff
@@ -0,0 +1,6 @@
+package net.dries007.tfc.objects.inventory.capability;
```

Comment:
This would make more sense in the `objects/fluids/capability` package I think.

Reply:
I just didn't know where to put it